src/logic/operate.js

```diff
@@ -0,0 +1,22 @@
+import Big from 'big.js';
+
+export default function operate(numberOne, numberTwo, operation) {
+  const one = Big(numberOne);
+  const two = Big(numberTwo);
+  if (operation === '+') {
+    return one.plus(two).toString();
+  }
+  if (operation === '-') {
+    return one.minus(two).toString();
+  }
+  if (operation === 'x') {
+    return one.times(two).toString();
+  }
+  if (operation === '÷') {
+    return one.div(two).toString();
+  }
```

Comment:
- [ ] Please work on the division of a number with zero let  us say `9/0` normally you have to get `error` in the screen but currently you are getting this
![shot](https://user-images.githubusercontent.com/44745855/133311273-df27645f-aa44-46ca-8498-ec0f0917f950.PNG)